af/fastpath/fastpath/utils.py

```diff
@@ -141,10 +141,35 @@
         },
     ],
     "IN": [
+        # https://github.com/ooni/pipeline/issues/25#issue-154919607
+        # https://explorer.ooni.org/measurement/7AEt2OwqdZUzFMzyZd3bFwnCwXp7FqYCezpaoBWdkBIfxtLtX84mXBZnlPLaTUqI?input=http:%2F%2Fthepiratebay.se
         {
             "body_match": "The page you have requested has been blocked",
             "locality": "country",
-        }
+        },
+        # TODO: maybe we would like to support defining a fingerprint by regexp?
```

Comment:
cc @FedericoCeratto

Reply:
It would be pretty expensive (and can potentially introduce vulnerabilities). Perhaps if we do the matching only on anomalies we could rethink of it?

Reply:
Yes you are right. I think it would actually be enough to also have a rule for headers which is `header_contains`. This would allow us to add a rule that is say:

```
"header_contains": "(Cisco-WSA/",
```

So that we can match all the Cisco-WSA equipment

Reply:
+1